packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -3,13 +3,26 @@ import { IWizardProps, IWizardStyles, IWizardStyleProps } from './Wizard.types';
 import { SubwayNav } from '../SubwayNav/SubwayNav';
 import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
 import { getStepToShow } from './Wizard.utils';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { wizardAnimationDurationMilliSec } from './Wizard.styles';
 
 const getClassNames = classNamesFunction<IWizardStyleProps, IWizardStyles>();
 
+// This returns a childFactory to provide to TransitionGroup
+// tslint:disable:no-any
+const childFactoryCreator = (classNames: any) => (child: any) =>
+  React.cloneElement(child, {
+    classNames
+  });
+
 /** Component for Wizard Base */
 export class WizardBase extends React.Component<IWizardProps, {}> {
+  private lastStepIndexShown: number;
+
   constructor(props: IWizardProps) {
     super(props);
+
+    this.lastStepIndexShown = 0;
```

Comment:
We should not just assign 0 here but the selected index or 0 if a selected index isn't set. Otherwise if we deep link into a Wizard (something we want to be able to do) you'd see a transition when we first come in because our conditional in our render would fire. (I think)